client/influxdb.go

```diff
@@ -145,6 +145,27 @@ func (c *Client) Ping() (time.Duration, string, error) {
 	return time.Since(now), version, nil
 }
 
+func (c *Client) Dump(db string) (*http.Response, error) {
```

Comment:
+1 for returning `io.ReadCloser`. It also doesn't tie the `Client.Dump()` interface to HTTP. Maybe we'll add a named pipes transport in the future. ;)